emailApp/voice/util.js

```diff
@@ -0,0 +1,48 @@
+import Tts from 'react-native-tts';
+
+export function checkStatus(state) {
+  const boolSend = state.sendEmail.includes('yes');
+  const addMore = state.recipient.includes('yes');
+
+  if (state.index === 0) {
+    Tts.speak(
+      'Welcome to this email sending app , I can help you in sending emails .  Do you want to send emails ?',
+    );
+  }
+
+  if (state.index === 1 && boolSend) {
```

Comment:
Conditions might not be needed, if we can store the same in an array for a particular index and fetch like 
```state[index]```
where state is an array and index starts from 0.